Assets/Scripts/UIManager.cs

```diff
@@ -0,0 +1,65 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+using UnityEngine.EventSystems;
+
+public class UIManager : MonoBehaviour
+{
+    public GameObject placement_object_prefab;
+
+    Ray mouse_ray;
+    RaycastHit hit;
+
+    GameObject tile;
+
+    GraphicRaycaster m_Raycaster;
```

Comment:
`m_Raycaster` ?? Big yikes my dude, it's obviously private.. no need to repeat yourself with `m_` - let's follow C# standards

Reply:
fixed